frontend/app/routes/protected/in-person/state-machine.ts

```diff
@@ -1,194 +1,143 @@
-import type { ActionFunctionArgs } from 'react-router';
-import { generatePath } from 'react-router';
+import { setup } from 'xstate';
 
-import type { Actor } from 'xstate';
-import { createActor, setup } from 'xstate';
-
-import { AppError } from '~/errors/app-error';
-import { ErrorCodes } from '~/errors/error-codes';
 import type { I18nRouteFile } from '~/i18n-routes';
-import { i18nRoutes } from '~/i18n-routes';
-import { getLanguage } from '~/utils/i18n-utils';
-import { getRouteByFile } from '~/utils/route-utils';
 
-type Event = { type: 'prev' } | { type: 'next' } | { type: 'cancel' };
-type Meta = { i18nRouteFile: I18nRouteFile };
+/**
+ * Type representing the possible state names for the in-person application process.
+ */
+export type StateName =
+  | 'start'
+  | 'privacy-statement'
+  | 'request-details'
+  | 'primary-docs'
+  | 'secondary-docs'
+  | 'name-info'
+  | 'personal-info'
+  | 'birth-info'
+  | 'parent-info'
+  | 'previous-sin-info'
+  | 'contact-info'
+  | 'review';
 
-const machineId = '(in-person-machine)';
+/**
+ * Mapping of state names to their corresponding route files.
+ * This object defines the routes for each state in the in-person application process.
+ * The keys are the state names, and the values are the paths to the route files.
+ */
+export const routes = {
+  'start': 'routes/protected/in-person/index.tsx',
+  'privacy-statement': 'routes/protected/in-person/privacy-statement.tsx',
+  'request-details': 'routes/protected/in-person/request-details.tsx',
+  'primary-docs': 'routes/protected/in-person/primary-docs.tsx',
+  'secondary-docs': 'routes/protected/in-person/secondary-docs.tsx',
+  'name-info': 'routes/protected/in-person/name-info.tsx',
+  'personal-info': 'routes/protected/in-person/personal-info.tsx',
+  'birth-info': 'routes/protected/in-person/birth-info.tsx',
+  'parent-info': 'routes/protected/in-person/parent-info.tsx',
+  'previous-sin-info': 'routes/protected/in-person/previous-sin-info.tsx',
+  'contact-info': 'routes/protected/in-person/contact-info.tsx',
+  'review': 'routes/protected/in-person/review.tsx',
+} satisfies Record<StateName, I18nRouteFile>;
```

Comment:
Set routes props as `readonly`

```suggestion
} as const satisfies Record<StateName, I18nRouteFile>;
```

Reply:
Good catch. I didn't have it originally because I wasn't exporting it.. I forgot to add it when exporting.

Reply:
07517daeb66a9aa407e7942dde2e3794a2bcb9ac